src/status_im2/contexts/onboarding/new_to_status/view.cljs

```diff
@@ -0,0 +1,79 @@
+(ns status-im2.contexts.onboarding.new-to-status.view
+  (:require
+   [quo2.components.markdown.text :as text]
+   [quo2.components.onboarding.small-option-card.view :as small-option-card]
+   [quo2.core :as quo]
+   [re-frame.core :as re-frame]
+   [react-native.core :as rn]
+   [status-im.keycard.recovery :as keycard]
+   [status-im.multiaccounts.recover.core :as multiaccounts.recover]
+   [status-im.react-native.resources :as resources]
+   [status-im2.contexts.onboarding.new-to-status.style :as style]))
+
+(defn background-image
+  []
+  [rn/image
+   {:style       style/image-background
+    :blur-radius 13
+    :source      (resources/get-image :onboarding-blur-bg)}])
+
+(defn navigation-bar
+  []
+  [rn/view {:style style/navigation-bar}
+   [quo/page-nav
+    {:align-mid?            true
+     :mid-section           {:type :text-only :main-text ""}
+     :left-section          {:type                :blur-bg
+                             :icon                :i/arrow-left
+                             :icon-override-theme :dark
+                             :on-press            #(re-frame/dispatch [:init-root :intro])}
+     :right-section-buttons [{:type                :blur-bg
+                              :icon                :i/info
+                              :icon-override-theme :dark
+                              :on-press            #(js/alert "Pending")}]}]])
+
+(defn sign-in-options
+  []
+  [rn/view {:style style/options-container}
+   [text/text
+    {:style  style/title
+     :size   :heading-1
+     :weight :semi-bold}
+    "I'm new to Status"]
```

Comment:
all copies should be in i18n

Reply:
I forgot to  ask about that! :facepalm: 

What's the process to add localized strings?
thanks!